ci/vm.sh

```diff
@@ -298,7 +306,7 @@ run_vm() {
 		-monitor unix:monitor.sock,server=on,wait=off \
 		-nographic \
 		"${serials[@]}" \
-		-machine $machine,accel=kvm:tcg -cpu $cpu -smp 2 -m 8192 \
+		-machine "$machine$kvm_accel" -cpu $cpu -smp 2 -m 8192 \
```

Comment:
Prefer "${machine}${kvm_accel}", particularly when variables get smashed up into each other: https://google.github.io/styleguide/shellguide.html#variable-expansion

I recognize that this file is all over the place for braces versus not braces.